surefire-its/src/test/java/org/apache/maven/surefire/its/JUnitPlatformRerunFailingTestsIT.java

```diff
@@ -43,22 +42,22 @@ private SurefireLauncher unpack()
     @Test
     public void testRerunFailingErrorTestsWithOneRetry() throws Exception
     {
-        OutputValidator outputValidator = unpack().addGoal( GOAL ).setJUnitVersion( VERSION ).maven().addGoal(
+        OutputValidator outputValidator = unpack().setJUnitVersion( VERSION ).maven().addGoal(
```

Comment:
Pls always use `debugLogging()` in every test. Sometimes this is the only way to understand the bug on the build system.